addons/weather/XEH_postServerInit.sqf

```diff
@@ -12,27 +12,21 @@ GVAR(ACE_rain) = rain;
 call FUNC(initWind);
 
 ["ace_settingsInitialized", {
-    TRACE_2("ace_settingsInitialized eh",GVAR(enableServerController),GVAR(serverUpdateInterval));
+    TRACE_3("ace_settingsInitialized eh",GVAR(enableServerController),GVAR(serverUpdateInterval),GVAR(useACEWeather));
 
     if (GVAR(enableServerController)) then {
         [FUNC(serverController), GVAR(serverUpdateInterval)] call CBA_fnc_addPerFrameHandler;
     };
 
     if (GVAR(useACEWeather)) then {
-        // Update rain every frame
-        addMissionEventHandler ["EachFrame", {0 setRain GVAR(ACE_rain)}];
+        GVAR(nextUpdateRain) = 0;
 
-        // Create a 1 sec delay PFEH to update rain
-        [{
-            BEGIN_COUNTER(weatherPFEHserver);
-
-            GVAR(nextUpdateRain) = 0;
+        addMissionEventHandler ["EachFrame", {
             if (CBA_missionTime >= GVAR(nextUpdateRain)) then {
```

Comment:
https://feedback.bistudio.com/T123355
And you are executing every 2 second? Why are you replicating what CBA PFH already does?